src/bin/compute-wer.cc

```diff
@@ -128,7 +128,8 @@ int main(int argc, char *argv[]) {
       }
       num_words += ref_sent.size();
       int32 ins, del, sub;
```

Comment:
I will avoid using bool as int.
This change is only for debugging purposes. The script finally uses align-text and uses that output.